src/test/kotlin/blackjack/GamblerTest.kt

```diff
@@ -20,4 +25,206 @@ class GamblerTest {
         gambler.addCard(cards)
         assertThat(gambler.isPlayerBelowBlackJack()).isFalse
     }
+
+    @Test
+    fun `return true if Player has two cards`() {
+        val gambler = Gambler(GamblerInfo("Player"))
+        val cards = listOf(Card(Rank.ACE, Suit.SPADE), Card(Rank.KING, Suit.SPADE))
+        gambler.addCard(cards)
+        assertThat(gambler.hasCardCount()).isTrue
+    }
```

Comment:
It would be great to add 'what returns true' in test name. In this case, I guess it would be `hasCardCount`.

Personally, I like to keep method names into test titles since it is easier to search for it, and it allows the team to automate various processes when managing a project.